From 8812321c2e979a62187e14abd7cc023bf93a1142 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sat, 29 Oct 2016 21:03:31 +0200
Subject: [PATCH 11/15] ARM: socfpga: ddr: Replace __FILE__ with __func__

The __FILE__ may contain a whole path in case the U-Boot is built
from within some buildsystem, which is useless and confusing, so
replace it with __func__.

Signed-off-by: Marek Vasut <marex@denx.de>
---
 drivers/ddr/altera/sequencer.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/ddr/altera/sequencer.c b/drivers/ddr/altera/sequencer.c
index 6c6bd90..39c6181 100644
--- a/drivers/ddr/altera/sequencer.c
+++ b/drivers/ddr/altera/sequencer.c
@@ -3534,7 +3534,7 @@ static void debug_mem_calibrate(int pass)
 	u32 debug_info;
 
 	if (pass) {
-		printf("%s: CALIBRATION PASSED\n", __FILE__);
+		printf("%s: CALIBRATION PASSED\n", __func__);
 
 		gbl->fom_in /= 2;
 		gbl->fom_out /= 2;
@@ -3553,7 +3553,7 @@ static void debug_mem_calibrate(int pass)
 		writel(debug_info, &phy_mgr_cfg->cal_debug_info);
 		writel(PHY_MGR_CAL_SUCCESS, &phy_mgr_cfg->cal_status);
 	} else {
-		printf("%s: CALIBRATION FAILED\n", __FILE__);
+		printf("%s: CALIBRATION FAILED\n", __func__);
 
 		debug_info = gbl->error_stage;
 		debug_info |= gbl->error_substage << 8;
@@ -3570,7 +3570,7 @@ static void debug_mem_calibrate(int pass)
 		writel(debug_info, &sdr_reg_file->failing_stage);
 	}
 
-	printf("%s: Calibration complete\n", __FILE__);
+	printf("%s: Calibration complete\n", __func__);
 }
 
 /**
@@ -3741,7 +3741,7 @@ int sdram_calibration_full(void)
 
 	initialize_tracking();
 
-	printf("%s: Preparing to start memory calibration\n", __FILE__);
+	printf("%s: Preparing to start memory calibration\n", __func__);
 
 	debug("%s:%d\n", __func__, __LINE__);
 	debug_cond(DLEVEL >= 1,
-- 
2.10.2

